packages/bff/src/routers/clientRouter.ts

```diff
@@ -6,48 +6,286 @@ import {
+          req.params.clientId,
+          ctx.authData.selfcareId,
+          ctx
+        );
+
+        return res.status(200).json(users).end();
+      } catch (error) {
+        const errorRes = makeApiProblem(
+          error,
+          getClientUsersErrorMapper,
+          ctx.logger
+        );
+        return res.status(errorRes.status).json(errorRes).end();
+      }
+    })
+
+    .post("/clients/:clientId/keys", async (req, res) => {
+      const ctx = fromBffAppContext(req.ctx, req.headers);
+      try {
+        await clientService.createKeys(
+          ctx.authData.userId,
+          req.params.clientId,
+          req.body,
+          ctx
+        );
+
+        return res.status(204).send();
+      } catch (error) {
+        const errorRes = makeApiProblem(error, emptyErrorMapper, ctx.logger);
+        return res.status(errorRes.status).json(errorRes).end();
+      }
+    })
+    .get("/clients/:clientId/keys", async (req, res) => {
+      const ctx = fromBffAppContext(req.ctx, req.headers);
+      try {
+        const keys = await clientService.getClientKeys(
+          req.params.clientId,
+          req.query.userIds,
+          ctx
+        );
+
+        return res.status(200).json({ keys }).end();
+      } catch (error) {
+        const errorRes = makeApiProblem(error, emptyErrorMapper, ctx.logger);
+        return res.status(errorRes.status).json(errorRes).end();
+      }
+    })
+
+    .get("/clients/:clientId/encoded/keys/:keyId", async (req, res) => {
+      const ctx = fromBffAppContext(req.ctx, req.headers);
+      try {
+        const key = await clientService.getEncodedClientKeyById(
+          req.params.clientId,
+          req.params.keyId,
+          ctx
+        );
+
+        return res.status(200).json(key).end();
+      } catch (error) {
+        const errorRes = makeApiProblem(error, emptyErrorMapper, ctx.logger);
+        return res.status(errorRes.status).json(errorRes).end();
+      }
+    })
+
+    .post("/clientsConsumer", async (req, res) => {
+      const ctx = fromBffAppContext(req.ctx, req.headers);
+
+      try {
+        const result = await clientService.createConsumerClient(req.body, ctx);
+
+        return res.status(200).json(result).end();
+      } catch (error) {
+        const errorRes = makeApiProblem(error, emptyErrorMapper, ctx.logger);
+        return res.status(errorRes.status).json(errorRes).end();
+      }
+    })
+
+    .post("/clientsApi", async (req, res) => {
+      const ctx = fromBffAppContext(req.ctx, req.headers);
+
+      try {
+        const result = await clientService.createApiClient(req.body, ctx);
+
+        return res.status(200).json(result).end();
+      } catch (error) {
+        const errorRes = makeApiProblem(error, emptyErrorMapper, ctx.logger);
+        return res.status(errorRes.status).json(errorRes).end();
+      }
+    })
+
     .get("/clients/:clientId/users/:userId/keys", async (_req, res) =>
       res.status(501).send()
     );
```

Comment:
Is the implementation of this route blocked by something?

Reply:
I was told that route will not be implemented becuase the frontend no longer needs it (we can double check on Slack maybe)

Reply:
https://pagopa.atlassian.net/browse/IMN-611?focusedCommentId=149731

Reply:
✅  maybe we could consider removing it from the API spec and the router if it's not supposed to by migrated?

Reply:
Yes, i can remove it so other reviewers can see it in the diff